src/main/resources/static/css/results.css

```diff
@@ -0,0 +1,10 @@
+.vb {
```

Comment:
For some reason, my GitHub Oauth redirects me to /css/results.css and loads this after logging in. We can look into this for a future PR maybe

Reply:
Yeah I noticed this as well! I'll add it to the issues

Reply:
#75 fixes this issue